cfme/utils/ssh.py

```diff
@@ -4,6 +4,8 @@
 from collections import namedtuple
 from subprocess import check_call
 
+import select
```

Comment:
**REQUIRED**
select is not used here

Reply:
Thanks...that was from a disappointing experiment.   Will remove it.